src/adera/Machines/RCSController.cpp

```diff
@@ -35,9 +35,10 @@ using Magnum::Vector3;
 
 
 MachineRCSController::MachineRCSController()
-    : Machine(true),
-    m_wiCommandOrient(this, "Orient"),
-    m_woThrottle(this, "Throttle")
+    : Machine(true)
+    , m_wiCommandOrient(this, "Orient")
```

Comment:
Put these member variable initializations in the class declaration, not the constructor.

That'll leave you with a 2 line constructor, which I would argue should live in the header file as an "inline".